src/app/child-dev-project/notes/model/note.ts

```diff
@@ -15,26 +15,38 @@
  *     along with ndb-core.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { MeetingNoteAttendance } from "../meeting-note-attendance";
 import { DatabaseEntity } from "../../../core/entity/database-entity.decorator";
 import { Entity } from "../../../core/entity/entity";
 import { DatabaseField } from "../../../core/entity/database-field.decorator";
 import { WarningLevel, WarningLevelColor } from "../../warning-level";
 import { InteractionType } from "../note-config-loader/note-config.interface";
+import { EventAttendance } from "../../attendance/model/event-attendance";
+import { AttendanceStatus } from "../../attendance/model/attendance-status";
+import { v4 as uuid } from "uuid";
 
 @DatabaseEntity("Note")
 export class Note extends Entity {
+  static create(date: Date, subject: string = ""): Note {
+    const instance = new Note(uuid());
```

Comment:
Is there a reason why you used `uuid` instead of `uniqid`? Anyway, maybe the `Entity` class can already make sure that the ID is set. I don't see a reason why the components/subclasses should be in charge of setting a correct ID.

Reply:
good point. I had somewhat forgotten about `uniqid`. uuid package support official (RFC standard) formats, which might guarantee more certainty to be unique but we should probably only use one.

Reply:
see PR #575